core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2244,6 +2244,14 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val EXECUTOR_ALLOW_SYNC_LOG_LEVEL =
+    ConfigBuilder("spark.executor.sync.log.level")
+      .doc("If set to true, log level applied through SparkContext.setLogLevel() method " +
+        "will be propagated to all executors.")
+      .version("3.5.0")
```

Comment:
@attilapiros Will this be merged into Spark 3.5.0? The current development branch is already at 4.0.0.

Reply:
@LuciferYang , Updated to `4.0.0`.